crates/polars-plan/src/plans/visitor/hash.rs

```diff
@@ -277,14 +277,14 @@ impl HashableEqLP<'_> {
                 IR::DataFrameScan {
                     df: dfl,
                     schema: _,
-                    output_schema: s_l,
+                    output_schema: _,
```

Comment:
NOTE: The output schema is used in none of the other branches. I don't see why it would be needed here